pkg/virt-handler/virtwrap/manager.go

```diff
@@ -137,7 +140,25 @@ type LibvirtDomainManager struct {
 	recorder record.EventRecorder
 }
 
+func (con LibvirtConnection) checkLibvirt() (done bool, err error) {
+	if virConn, err := newConnection(con.uri, con.user, con.pass); err == nil {
+		defer virConn.Close()
```

Comment:
Could you add a log message somewhere here? Not sure about the verbosity level, but I would want to have a way to see that it is currently waiting for libvirt.